<launch>
  <node pkg="rubot_projects" type="rubot_project1_picture.py" name="take_photo" output="screen">
    <param name="image_topic" value="/usb_cam/image_raw"/>
    <param name="image_title" value="$(find rubot_projects)/photos/photo3_hw_launch_2.jpg"/>
  </node>
</launch>